src/config.c

```diff
@@ -2527,6 +2527,7 @@ standardConfig configs[] = {
     createSizeTConfig("hll-sparse-max-bytes", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.hll_sparse_max_bytes, 3000, MEMORY_CONFIG, NULL, NULL),
     createSizeTConfig("tracking-table-max-keys", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.tracking_table_max_keys, 1000000, INTEGER_CONFIG, NULL, NULL), /* Default: 1 million keys max. */
     createSizeTConfig("client-query-buffer-limit", NULL, MODIFIABLE_CONFIG, 1024*1024, LONG_MAX, server.client_max_querybuf_len, 1024*1024*1024, MEMORY_CONFIG, NULL, NULL), /* Default: 1GB max query buffer. */
+    createSizeTConfig("maxmemory-clients", NULL, MODIFIABLE_CONFIG, 0, ULLONG_MAX, server.maxmemory_clients, 0, MEMORY_CONFIG, NULL, NULL),
```

Comment:
Should the minimum value be set to a larger value, too small a value may cause the user to be disconnected as soon as connected.

Reply:
Currently a value of 0 means no limit. It makes sense to have a lower bound (something like 1MB), but then how will we handle the special case of no-limit?

Reply:
Or add a policy? like```maxmemory-policy```.
```client_max_querybuf_len``` Will it also be deprecated? 
Maybe user only wants to limit query, not output, I can set the policy to ```volatile-client-query```.